source/utilities.tex

```diff
@@ -5111,12 +5111,13 @@
 The non-member \tcode{any_cast} functions provide type-safe access to the contained value.
 
 \pnum
-Implementations should avoid the use of dynamically allocated memory for a small contained value.
+Implementations should avoid the use of dynamically allocated memory for a small contained value;
+any such small-object optimization shall only be applied to types \tcode{T} for which
```

Comment:
A connective such as "however" would make more sense than a semicolon here, as we are partially retracting the advice given in the previous clause.

Reply:
Fixed.